index.bs

```diff
@@ -251,15 +253,15 @@ The semantics of the clean aperture property (<code>'[=clap=]'</code>) as define
 
 <h4 id="other-item-property">Other Item Properties</h4>
 
-In addition to the Image Properties defined in this document, [=AV1 image items=] MAY also be associated with item properties defined in other specifications such as [[!HEIF]] and [[!MIAF]]. Examples of commonly used item properties are:
-    - <code>'[=colr=]'</code>
-    - <code>'[=pixi=]'</code>
+In addition to the Image Properties defined in this document, [=AV1 image items=] may also be associated with item properties defined in other specifications such as [[!HEIF]] and [[!MIAF]]. Examples of commonly used item properties are:
+    - <code>'[=colr=]'</code> (<dfn noexport for="AVIF">ColourInformationBox</dfn>)
```

Comment:
I feel it adds indirection. When one wants to look `ColourInformationBox` up, it will first lead to a place in AVIF, which itself redirects to HEIF. I get that this behavior is wanted to map `colr` to `ColourInformationBox`, but it also introduces confusion about which spec defines what.

> It's very hard to figure out how the various specs fit together.

I do not see an easy solution to this.

> It's unclear which things in HEIF and MIAF are actually supported/allowed in AVIF.

Solutions I can think of:
- Rely on the fact that everything that is not explicitly forbidden is allowed (~ current situation, could be written somewhere though)
- Succinctly list all boxes or features that are allowed (kind of an extended section 9. Box requirements) and declare all others as forbidden
- Create a new section for each supported box or feature, as was done for `tmap` and now `grid` etc. This feels like copy/pasting parts of ISOBMFF/HEIF/MIAF, but may be the clearest way for someone that just reads AV1-AVIF and not the inherited specs.

Example for `iden` which is allowed by AV1-AVIF but barely supported in implementations:
- Not explicitly forbidden hence allowed,
- Could be listed as `<code>[=iden=]</code>` and that's it,
- There could be a whole section about it.

Not sure which one is better than the others to be honest.

Anyway this seems beyond the topic of `AVIF/ColourInformationBox` and `AVIF/ColourInformationBox` barely answers that lack of clarity in my opinion.

I do not mind using both terms directly linked to 14496-12, especially if both are next to each other in the `dfn` list, but what about sticking to one of `colr` and `ColourInformationBox` if that is too much?

Reply:
> Not sure which one is better than the others to be honest.

Yeah, that's what I was grappling with as well.

> I do not mind using both terms directly linked to 14496-12, especially if both are next to each other in the dfn list, but what about sticking to one of colr and ColourInformationBox if that is too much?

In order to improve consistency, how about this:
1. We switch all of the current usage of the full HEIF box names in AVIF to use the box types. That's mostly what we are using in any case, so it will only be a few changes. All of the box types that don't have explicit AVIF requirements or text links directly to the spec that defines it.
2. We add to the AVIF spec the boxes which are reasonably widely supported or something that we believe may end up being common in the foreseeable future. This means:
    1. For derived images we keep the sections for `grid` and `tmap` since they are widely supported. Since `iden` and `iovl` is only supported on Apple systems (I think?) it doesn't count as widely supported so we don't add sections for those. All use of `grid` and `tmap` inside of AVIF link to the AVIF sections which in turn links to HEIF.
    2. For item properties we add the properties that are widely supported or at least likely to be used to the list of boxes in `avif-required-boxes-additional`. 
    3. We either add common properties to the section "Other Item Properties" or completely remove that section and point it to `avif-required-boxes-additional`. I'm leaning towards the latter since it feels like we're duplicating stuff.

If we all agree on that, I guess the question comes down to which boxes to keeps in the list of boxes. 
- I added the `grpl`, `altr` and `ster` boxes since the first two are needed for `tmap` items and `ster` is pretty much the base requirement for stereo photos and we use them extensively for HEIC.
- I added all of the HDR item properties that have been added since they seem like they will likely be used now or in the future (and they are used in SlimHEIF).
- I also added the camera extrinsics and intrinsics since those are very important for stereo images and supported by libheif and the Apple platforms.

Reply:
> 1. We switch all of the current usage of the full HEIF box names in AVIF to use the box types

So `colr` everywhere and no mention of `ColourInformationBox`. This is fine with me, although I would have picked the latter, personally.

> 2. We add to the AVIF spec the boxes which are reasonably widely supported

Sounds good to me, as long as it is explicit enough that they are not _defined_ by AVIF.

> 2.iii. completely remove "Other Item Properties" section

Agreed.

> which boxes to keep

I trust you on that. I am not that knowledgeable on what is used out there.

Reply:
Yes, I meant switching all, `colr` was just an example.

Reply:
Actually, looking through the doc, we use the box names consistently in some cases (`FileTypeBox`). Let me handle this as follows instead:
For the cases where we _mostly_ refer to the box by the 4CC but at some point use the name, make it look like this:
```
<code>[=MasteringDisplayColourVolumeBox=]</code> (<code>'[=mdcv=]'</code>)
```

Reply:
@y-guyon 
Let me know what you think. I've now made everything link directly to the base spec, _except in these places_:
1. `clap` in table of boxes links to `#clean-aperture-property` section
2. `grpl` in table of boxes links to `#groups` section
3. `altr` in table of boxes lins to `#altr-group` section
4. `ster` in table of boxes lins to `#ster-group` section

The places where we mention using `altr` groups now say something like this:
```
should be grouped together by an <code>'[=altr=]'</code> (see [[#altr-group]])
```

This way we link to the base spec _and_ add a link to the relevant section in AVIF.

Reply:
Sorry, I missed your replies. This looks good, thank you for the changes.